include/cute/atom/copy_traits_xe.hpp

```diff
@@ -557,6 +557,24 @@ struct Copy_Traits_<XE_2D_Packed_U8x1x32_LD_N, args_t...>
       : XE_2D_LD_Unpack<XE_2D_Packed_U8x1x32_LD_N, args_t...>(args...) {}
 };
 
+template <class... args_t>
+struct Copy_Traits_<XE_2D_U8x1x32_LD_N, args_t...>
+    : XE_2D_LD_Unpack<XE_2D_U8x1x32_LD_N, args_t...> {
+  using ThrID = Layout<_16>;
+  // Map from (src-thr,src-val) to bit
+  using SrcLayout = Layout<Shape <_16,_16>,
+                           Stride< _0,_1>>;
+  // Map from (dst-thr,dst-val) to bit
+  using DstLayout = Layout<Shape <_16,_16>,
+                           Stride<_16, _1>>;
```

Comment:
You are setting the layouts the same as above one - is this the same operation?

I am guessing you want this here:
```suggestion
  using SrcLayout = Layout<Shape <_16,Shape<_8,_2>>,
                           Stride< _0,Stride<_1,_128>>>;
  // Map from (dst-thr,dst-val) to bit
  using DstLayout = Layout<Shape <_16,Shape<_8,_2>>,
                           Stride<_8, Stride<_1,_128>>>;
```

Reply:
yes, because this layout are same semantics with above one, so i don't change it. your change is more accurate。